tsconfig.json

```diff
@@ -22,6 +22,7 @@
         },
         "lib": [
             "es2015",
+            "es2017",
```

Comment:
Can `es2015` then be removed?

Reply:
They can both be replaced by `'esnext'`, because that is also the `target` above anyway.